lib/components/app/app-nav.js

```diff
@@ -4,26 +4,33 @@ import { connect } from 'react-redux'
 
 import NavLoginButtonAuth0 from '../user/nav-login-button-auth0.js'
 import { accountLinks, getAuth0Config } from '../../util/auth'
-
 import AppMenu from './app-menu'
 
-const AppNav = ({ otpConfig }) => {
+const AppNav = ({ otpConfig, title }) => {
```

Comment:
This is only intended for Desktop use, right? Please add jsdoc to that effect and/or rename to DesktopNavbar.

Reply:
I choose `DesktopNavBar`.